src/game/models/mod.rs

```diff
@@ -2,156 +2,16 @@
 
 use std::f64;
```

Comment:
Oh yeah, that is true. Maybe it doesn't care because it's going to be linked into the final executable anyway?